browser/src/GenePage/VariantCooccurrenceCountsTable.spec.tsx

```diff
@@ -7,13 +7,58 @@ import {
 } from '../__factories__/VariantCooccurrenceCountsPerSeverityAndAf'
 
 import VariantCooccurrenceCountsTable from './VariantCooccurrenceCountsTable'
+import { forDatasetsMatching, forDatasetsNotMatching } from '../../../tests/__helpers__/datasets'
+
+const v2Regexp = /_r2/
+
+forDatasetsNotMatching(
```

Comment:
Not worth delaying this change for, but at some convenient point we should define a couple of constants like this in a test helper library.